NEWS.md

```diff
@@ -1,4 +1,14 @@
-## 2.1.0 - Unreleased
```

Comment:
Update please NEWS.md with bug fixes from long leave branch.

Reply:
Updated, thank you.